app/views/rules_of_origin/steps/_about_commodity.html.erb

```diff
@@ -0,0 +1,15 @@
+<%= render 'shared/details', summary: t('.summary') do %>
+  <%# full key necessary for i18n because we're inside shared/details %>
```

Comment:
maybe remove this comment?

Reply:
I'd left it there for future developers - if they use abbreviated keys then they'll probably not realise why they don't work - since they need to be under `shared.details` instead of `rules_of_origin.steps.about_commodity` in the `en.yml`?